### PR TITLE
Minor: Fixes to Selector's javadoc

### PR DESCRIPTION
**Pull request by ijuma**

### CODE REVIEW

**Comment by ijuma** [comment]
Review by @guozhangwang and/or @hachikuji

File: clients/src/main/java/org/apache/kafka/common/network/Selector.java [hunk 1]
```diff
@@ -45,7 +45,7 @@
  * A connection can be added to the nioSelector associated with an integer id by doing
  * 
  * <pre>
- * nioSelector.connect(42, new InetSocketAddress(&quot;google.com&quot;, server.port), 64000, 64000);
+ * nioSelector.connect(&quot;42&quot;, new InetSocketAddress(&quot;google.com&quot;, server.port), 64000, 64000);
```
Comment (hachikuji) [inline clients/src/main/java/org/apache/kafka/common/network/Selector.java:48]: Why did the number 42 need to be quoted?

Comment (granthenke) [inline clients/src/main/java/org/apache/kafka/common/network/Selector.java:48]: Looks like nioSelector.connect takes a String not an Int.

Comment (hachikuji) [inline clients/src/main/java/org/apache/kafka/common/network/Selector.java:48]: Oh, nevermind. I didn't notice the API change.

Comment (ijuma) [inline clients/src/main/java/org/apache/kafka/common/network/Selector.java:48]: Indeed.

**Comment by hachikuji** [comment]
@ijuma LGTM

File: clients/src/main/java/org/apache/kafka/common/network/Selector.java (hunk 1 above)
Comment (gwenshap) [inline clients/src/main/java/org/apache/kafka/common/network/Selector.java:48]: Thanks for fixing. I changed the API and forgot the doc. Selector used to take a broker-id as identifier when it was used just for client-broker communications. Once we needed it for inter-broker, we needed more flexibility and changed that to String.

**Pull request closed without merging**